packages/eslint-plugin/docs/rules/explicit-function-return-type.md

```diff
@@ -84,6 +84,30 @@ node.addEventListener('click', function() {});
 const foo = arr.map(i => i * i);
 ```
 
+### allowTypedFunctionExpressions
+
+Examples of **incorrect** code for this rule with `{ allowTypedFunctionExpressions: true }`:
+
+```ts
+let arrowFn = () => 'test';
+
+let funcExpr = function() {
+  return 'test';
+};
+```
+
+Examples of **correct** code for this rule with `{ allowTypedFunctionExpressions: true }`:
```

Comment:
How do you feel about removing the “incorrect” section above and changing this to say “Examples of additional **correct** code…”?

Reply:
Sounds lighter. I like it.